tutorials/demo/Matplotlib_Figures.ipynb

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
<p>I think almost all of the code we're going to see will use the imperative style - I like that you say <strong>don't mix</strong> rather than <strong>pick OO always</strong>; the latter is a bit of an uphill battle.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuromatchAcademy/course-content/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>